src/Raven.Server/Documents/Queries/Parser/QueryParser.cs

```diff
@@ -1319,7 +1319,23 @@ private DeclaredFunction DeclaredFunction()
                 // validate this function
                 try
                 {
+                    var p1 = Scanner.Position;
+                    var x = Scanner.Position - funcBodyStart - 2;
```

Comment:
lets have better naming, something like:
```suggestion
                    var pos = Scanner.Position;
                    var includePos = Scanner.Position - funcBodyStart - 2;
```

Reply:
done